CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+
+### v3.4.2
+
+**Fixed**
+- Fix clair crash downloading RHEL content mapping
+- Quay config validates SMTP
+- Quay Operator mirror configurations cannot be decoded after QuayRegistry
+reconciliation
+- Fix encrypted passwords
```

Comment:
"config-tool now prevents `SECRET_KEY` from changing on config updates"

Reply:
There wasn't a bug with encrypted passwords. The customer was not URL encoding their credentials with special characters. Closed the issue as "not a bug" so I don't think this needs to be in the changelog.

Reply:
@thomasmckay @jonathankingfc was there any actual change with SECRET_KEY in the config tool? If not we should just remove this.